streams/eg.m3u

```diff
@@ -32,12 +32,14 @@ https://5d658d7e9f562.streamlock.net/koogi.tv/koogi.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="MekameleenTV.eg",Mekameleen TV (1080p)
 https://mn-nl.mncdn.com/mekameleen/smil:mekameleentv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="NogoumFMTV.eg",NogoumFMTV (672p) [Not 24/7]
-https://nogoumtv.nrpstream.com/hls/stream.m3u8
+https://nogoumtv.nrpstream.com/hls/0/stream.m3u8
```

Comment:
Basically avoiding a redirection. If they decide to upgrade their quality, it's better to stay with the adaptive stream.